t/067-req-socket.t

```diff
@@ -1096,3 +1096,73 @@ done
 --- grep_error_log_out
 lua finalize socket
 GC cycle done
+
+
+
+=== TEST 18: receiveany
+--- config
+    location = /t {
+        content_by_lua '
```

Comment:
One more thing, could you use `content_by_lua_block` instead of the deprecated `content_by_lua` directive in the new test cases? So we don't need to pay too much for the technical debt.

Reply:
Yup, done